sandbox/vite-ts/src/App.tsx

```diff
@@ -1,13 +1,37 @@
 import { css, cx } from '../styled-system/css'
-import { Circle, HStack, Stack, panda } from '../styled-system/jsx'
+import { Circle, HStack, Stack, isCssProperty, panda } from '../styled-system/jsx'
 import { circle as circleLike, stack, vstack } from '../styled-system/patterns'
 import { button, someRecipe } from '../styled-system/recipes'
 import { Badge, badge } from './Badge'
 import { AnotherButtonWithRegex, Button, ListedButton } from './Button'
 import { Card } from './Card'
+import { motion, isValidMotionProp } from 'framer-motion'
 
 const SomeRecipe = panda('div', someRecipe)
 
+const StyledMotion = panda(
+  motion.div,
+  {},
+  {
+    /*  Allow motion props and non-style props to be forwarded. */
+    shouldForwardProp: (prop, variantKeys) =>
+      isValidMotionProp(prop) || (!variantKeys.includes(prop) && !isCssProperty(prop)),
+  },
+)
```

Comment:
For the types to work correctly, I think we should do the reverse. Create a motion component from a styled.div

```jsx
const StyledMotion = motion(styled.div)
```

This might make the example not useful tbh, but we can use a different example.